plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -483,4 +485,30 @@ public IcebergConfig setMetadataCacheEnabled(boolean metadataCacheEnabled)
         this.metadataCacheEnabled = metadataCacheEnabled;
         return this;
     }
+
+    @Config("iceberg.object-store.enabled")
+    @ConfigDescription("Enable the Iceberg object store file layout")
+    public IcebergConfig setObjectStoreEnabled(boolean objectStoreEnabled)
+    {
+        this.objectStoreEnabled = objectStoreEnabled;
+        return this;
+    }
+
+    public boolean isObjectStoreEnabled()
+    {
+        return objectStoreEnabled;
+    }
+
+    @Config("iceberg.data-location")
```

Comment:
The new properties are not documented in `iceberg.md`. Please consider doing so once we've settled whether the PR actually adds value to `trino-iceberg` connector.

Do consider to explain there whether there is any correlation between the two new config properties.

Reply:
@findinpath Added the docs